### PR TITLE
GUI improvements

### PR DESCRIPTION
**Pull request by odeimaiz**
- 4 templates: 2 Colleen Clancys, Demo December and NIH Hackathon 2018
- speag.swiss included in inHouse list
- getUserName method added (from email)
- ProjectModel initialization improved
- Project Browser:
  - smaller thumbnails
  - scrollbar added

Everything fits better now in a smaller screen
image

### CODE REVIEW

File: services/web/client/source/class/qxapp/data/Store.js [hunk 1]
```diff
@@ -50,7 +50,8 @@ qx.Class.define("qxapp.data.Store", {
     getRole: function() {
       // Temporary HACK
       const userEmail = qxapp.auth.Data.getInstance().getEmail();
-      if (userEmail.includes("itis.swiss") || userEmail.includes("oetiker.ch")) {
+      const inHouse = ["itis.swiss", "speag.swiss", "oetiker.ch"];
```
Comment (pcrespov) [inline services/web/client/source/class/qxapp/data/Store.js]: mmhhh I would not hard-code this. Can u pass it as a configuration variable upon compilation?

File: services/web/server/src/simcore_service_webserver/projects_fakes.py [hunk 2]
```diff
@@ -50,9 +50,11 @@ def load_user_projects(cls, user_id=None):
     @classmethod
     def load_template_projects(cls):
         templateFile = "data/fake-template-projects.json"
-        templateFile = "data/fake-template-projects.osparc.json"
         with resources.stream(templateFile) as f:
             projects = json.load(f)
+        templateOsparcFile = "data/fake-template-projects.osparc.json"
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/projects_fakes.py]: please use snake_case in python

File: services/web/client/source/class/qxapp/data/Store.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/qxapp/data/Store.js]: This is just a temporary hack until the user resource gets in, I wouldn't overdo it

File: services/web/server/src/simcore_service_webserver/projects_fakes.py (hunk 2 above)
Comment (odeimaiz) [reply services/web/server/src/simcore_service_webserver/projects_fakes.py]: okey dokey

File: services/web/client/source/class/qxapp/data/Store.js (hunk 1 above)
Comment (pcrespov) [reply services/web/client/source/class/qxapp/data/Store.js]: please add a TODO or FIXME

File: services/web/client/source/class/qxapp/data/Store.js [hunk 3]
```diff
@@ -50,7 +50,8 @@ qx.Class.define("qxapp.data.Store", {
...
+      if (inHouse.some(v => userEmail.indexOf(v) >= 0)) {
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/data/Store.js]: your soooo old style :)

how abot using and object/hash or regexp maching for this kind of test ?

**Pull request merged**